src/ModulesFactory.cpp

```diff
@@ -24,6 +25,12 @@ ::mesos::slave::Isolator* createIsolator(
                              cfg.cleanupCommand, cfg.usageCommand,
                              cfg.isDebugSet);
 }
+
+::mesos::slave::ResourceEstimator* createResourceEstimator(
+    const ::mesos::Parameters& parameters) {
+  Configuration cfg = ConfigurationParser::parse(parameters);
+  return new OpportunisticResourceEstimator(cfg.prepareCommand, cfg.isDebugSet);
```

Comment:
why is it named prepareCommand? It should be something related to what the binary does (something like resourceEstimatorCommand)

Reply:
Good point i have changed the name to **CommandResourceEstimator** to avoid confusion with **mesos::ResourceEstimator**